src/js/App/GlobalFilter/GlobalFilter.js

```diff
@@ -9,7 +9,7 @@ import { Chip, ChipGroup } from '@patternfly/react-core/dist/js/components/ChipG
 import { Button } from '@patternfly/react-core/dist/js/components/Button';
 import { Tooltip } from '@patternfly/react-core/dist/js/components/Tooltip';
 import TagsModal from './TagsModal';
-import { workloads, updateSelected, storeFilter, generateFilter, selectWorkloads } from './constants';
+import { workloads, updateSelected, storeFilter, generateFilter } from './constants';
```

Comment:
I don't see the `selectWorkloads` to be used anywhere else. Can we delete the function declaration as well?

Reply:
Already removed https://github.com/RedHatInsights/insights-chrome/pull/1119/files#diff-108371adf9416c27a728f9b30a115850331944eaf03f26ab052c7ade2447bd93L26